### PR TITLE
formatting 3/n: PL modules

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

apply formatting to some more packages

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by justusschock — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Review by ydcjeff — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]

File: pytorch_lightning/callbacks/early_stopping.py [hunk 1]
```diff
@@ -121,9 +117,11 @@ def __init_monitor_mode(self):
     def _validate_condition_metric(self, logs):
         monitor_val = logs.get(self.monitor)
 
-        error_msg = (f'Early stopping conditioned on metric `{self.monitor}`'
-                     f' which is not available. Pass in or modify your `EarlyStopping` callback to use any of the'
-                     f' following: `{"`, `".join(list(logs.keys()))}`')
+        error_msg = (
+            f'Early stopping conditioned on metric `{self.monitor}`'
```
Comment (kaushikb11) [inline pytorch_lightning/callbacks/early_stopping.py:121]: There are single quotes being used at some places?

Comment (awaelchli) [reply pytorch_lightning/callbacks/early_stopping.py:121]: yeah @Borda I also thought the formatter would standardize that?

Comment (Borda) [reply pytorch_lightning/callbacks/early_stopping.py:121]: seems that yapf doe not force string, we may ask it as a feature :]

**Review by akihironitta — APPROVED** [review]

**Review by rohitgr7 — APPROVED** [review]

File: pytorch_lightning/loggers/base.py [hunk 2]
```diff
@@ -95,9 +98,9 @@ def update_agg_funcs(
     def experiment(self) -> Any:
         """Return the experiment object associated with this logger."""
 
-    def _aggregate_metrics(
-            self, metrics: Dict[str, float], step: Optional[int] = None
-    ) -> Tuple[int, Optional[Dict[str, float]]]:
+    def _aggregate_metrics(self,
+                           metrics: Dict[str, float],
+                           step: Optional[int] = None) -> Tuple[int, Optional[Dict[str, float]]]:
```
Comment (rohitgr7) [inline pytorch_lightning/loggers/base.py:103]: I wonder why isn't it formatted like this:
```python
def fun(
    arg1,
    arg2,
)
```
just like others.

File: pytorch_lightning/profiler/profilers.py [hunk 3]
```diff
@@ -396,11 +397,13 @@ def __init__(
...
             rank_zero_warn(
                 "The exported trace would be save locally as `path_to_export_trace` is empty."
-                " Note: Each functions will generate its own traced file.")
+                " Note: Each functions will generate its own traced file."
+            )
 
         if self.sort_by_key not in self.AVAILABLE_SORT_KEYS:
             raise MisconfigurationException(
-                f"Found sort_by_key: {sort_by_key}. Should be within {self.AVAILABLE_SORT_KEYS}. ")
+                f"Found sort_by_key: {sort_by_key}. Should be within {self.AVAILABLE_SORT_KEYS}. "
```
Comment (rohitgr7) [inline pytorch_lightning/profiler/profilers.py:405]:
```suggestion
                f"Found sort_by_key: {sort_by_key}. Should be within {self.AVAILABLE_SORT_KEYS}."
```

**Review by rohitgr7 — APPROVED** [review]

File: pytorch_lightning/loggers/base.py (hunk 2 above)
Comment (Borda) [reply pytorch_lightning/loggers/base.py]: well in this case I would rather expect as they are short
```
def fun(
    arg1, arg2,
) -> ... :
```
but fixed with , at the end

**Review by awaelchli — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

**Pull request merged**